flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java

```diff
@@ -70,7 +70,7 @@ public static void startNonSSLServer() throws IOException {
 		config.setString(BlobServerOptions.STORAGE_DIRECTORY, temporarySslFolder.newFolder().getAbsolutePath());
 		config.setBoolean(BlobServerOptions.SSL_ENABLED, false);
 
-		blobNonSslServer = new BlobServer(config, new VoidBlobStore());
+		blobNonSslServer = new TestBlobServer(config, new VoidBlobStore());
```

Comment:
`BlobClientSslTest` inherits `BlobClientTest`, and it will also execute the new test `testSocketTimeout` added in `BlobClientTest`. If `TestBlobServer` is not used here, an error will occur.

Reply:
I checked the code carefully, and `TestBlobServer`  is not needed here. I will revert to `BlobServer`.